lib/utils/base_env.dart

```diff
@@ -33,7 +41,7 @@ String get envLcdPort => const String.fromEnvironment('LCD_PORT', defaultValue:
 
 // DO NOT USE String.fromEnvironment without 'const'!!!
 // https://github.com/flutter/flutter/issues/55870#issuecomment-620776138
-String get envGrpcPort => const String.fromEnvironment('GRPC_PORT_', defaultValue: '9090');
+String get envGrpcPort => const String.fromEnvironment('GRPC_PORT', defaultValue: '9090');
```

Comment:
a typo in `GRPC_PORT` for --dart-define

Reply:
Yeah i did ths intentionally to force it to use use the fallback value